lib/dal/src/func.rs

```diff
@@ -37,15 +37,12 @@ pub mod summary;
 pub mod view;
 
 mod associations;
-mod before;
```

Comment:
Move before funcs to the secrets domain so that everything related to secrets is co-located. This PR does this for tests too.